src/index.ts

```diff
@@ -1,7 +1,7 @@
 export * as promises from './promises';
 export * as logging from './logging';
 export * from './types';
-export * as opsGenie from './opsgenie';
+export * from './opsgenie';
```

Comment:
Not so sure about this, it makes imports downstream pretty, I should probably do it for everything/one namespace.